.changeset/breezy-berries-play.md

```diff
@@ -0,0 +1,5 @@
+---
+"wrangler": minor
+---
+
+Add commands for adding & deleting event notification configurations
```

Comment:
Oh actually could you fill this out a bit more. It is what will appear in the Wrangler changelog. For example, this doesn't mention that this is for R2 commands.

Reply:
Updated to:

> Add `event-notification` commands in support of event notifications for Cloudflare R2.

> Included are commands for creating and deleting event notification configurations for individual buckets.